### PR TITLE
Update NuGet packages

### PR DESCRIPTION
**Pull request by NetDwarf**
Add netstandard.dll to referenced assemblies in ScriptMgr
Remove "Ignore Prepare" from MySQL ConnectionString

### CODE REVIEW

**Comment by NetDwarf** [comment]
Special thanks to @OntheDOL for testing MySql.Data package :smiley:

File: DOLConfig/DOLConfig.csproj [hunk 1]
```diff
@@ -67,8 +67,8 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="System.Windows.Forms" />
-    <PackageReference Include="MySql.Data" Version="6.10.9" ExcludeAssets="Runtime" />
-    <PackageReference Include="System.Data.SQLite.Core" Version="1.0.113.6" ExcludeAssets="All" Condition=" '$(OS)' != 'Unix' " />
+    <PackageReference Include="MySql.Data" Version="8.0.25" ExcludeAssets="Runtime" />
```
Comment (Lotendan) [inline DOLConfig/DOLConfig.csproj:70]: Glad to see they finally fixed the performance regression in https://bugs.mysql.com/bug.php?id=101714

**Review by Lotendan — APPROVED** [review]

**Comment by Lotendan** [comment]
Hello, any info regarding my question above please?
Thanks ;)

**Comment by NetDwarf** [comment]
Sorry, I oversaw the question. It sounds interesting, however there is no plan to change MySql.Data since it is working fine for DOL purposes given it is not that data heavy.

**Pull request merged**